go/service/teams.go

```diff
@@ -241,10 +241,13 @@ func (h *TeamsHandler) TeamAcceptInvite(ctx context.Context, arg keybase1.TeamAc
 	if err := h.assertLoggedIn(ctx); err != nil {
 		return err
 	}
+
+	// If token is valid seitan, don't pass to functions that might log or send to server.
```

Comment:
I think, applying your argument, you should be even more cautious. If the token has a `+` in it, then consider it a Seitan token, *even* if the wrong number of characters.  This way you don't send seitan tokens off to the server if there's an extra or dropped character.